docs/lessons/10-interfaces-and-abstract-classes.md

```diff
@@ -0,0 +1,47 @@
+# 10 - Interfaces & Abstract Classes
+
+<Teacher name="Faiçal"></Teacher>
+
+## Learning Objectives
+
+At the end of the class, attendees should be able to
+
+- Define what an interface is
+- Create and use interfaces in Java
+- Explain what interfaces are useful for
+- Define what an abstract class is
+- Create and use abstract classes in Java
+- Explain what abstract classes are useful for
+- Explain the differences between abstract classes and interfaces
+
+## Presentation
+
+[Interfaces & Abstract Classes](https://speakerdeck.com/ftchirou/interfaces-and-abstract-classes-in-java)
+
+## Exercices
+
+### Netflix, continued
+
+Review the [Netflix task](https://redi-j2.netlify.com/lessons/09-interitance.html#inheritance) from the [previous lesson](https://redi-j2.netlify.com/lessons/09-interitance.html) and re-implement it **without using inheritance**. Use only the concepts studied in this lesson.
+
+Below are the main requirements to keep in mind when working through this task.
+
+- A `user` has a subscription `plan` on Netflix.
+- A `user` can watch a `movie` on Netflix.
+- A `movie` has a `title`.
+- Netflix offers 3 different plans `Basic`, `Standard` and `Premium`. Each plan has some characteristics, for example: 
+    - the  monthly price
+    - the number of `devices` the user can watch on at the same time
```

Comment:
maybe some task with device as well, so they can make it as Interface ?

Reply:
Good point 👌🏾